bundles/org.eclipse.swt/Eclipse SWT/gtk/org/eclipse/swt/widgets/Table.java

```diff
@@ -199,6 +199,9 @@ long cellDataProc (long tree_column, long cell, long tree_model, long iter, long
 	int [] index = new int [1];
 	C.memmove (index, GTK.gtk_tree_path_get_indices (path), 4);
 	TableItem item = _getItem (index[0]);
+	if (item.isDisposed()) {
+		return 0;
```

Comment:
do we still need to call `GTK.gtk_tree_path_free (path);` here?

Reply:
Good catch. https://github.com/eclipse-platform/eclipse.platform.swt/pull/2473 should fix it.